### PR TITLE
SE-2801: Transaction Consolidation NB

### PR DESCRIPTION
**Pull request by Tomwp**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Added a notebook describing how transaction types from different sources can be consolidated and harmonized into a single taxonomy.

### CODE REVIEW

File: examples/use-cases/ibor/Transaction Type Consolidation.ipynb [hunk 1]
```diff
@@ -0,0 +1,1239 @@
+{
```
Comment (jyaffe) [inline examples/use-cases/ibor/Transaction Type Consolidation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #9.    instruments
</pre><p>probably worth adding 'transaction types' here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/556/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (jyaffe) [inline examples/use-cases/ibor/Transaction Type Consolidation.ipynb:1]: <p>You show it in the code later but is it worth also saying here that the main transaction type value returned by LUSID will be <code>Buy</code> but the original input type can be accessible via the default system property <code> Transaction/default/TxnInputType.value.label_value</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/556/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (Tomwp) [reply examples/use-cases/ibor/Transaction Type Consolidation.ipynb:1]: Added.

Comment (Tomwp) [reply examples/use-cases/ibor/Transaction Type Consolidation.ipynb:1]: Added in. Also added a link to section 5.1.

**Comment by jyaffe** [comment]
LGTM - probably worth getting a second pair of eyes

File: examples/use-cases/ibor/Transaction Type Consolidation.ipynb (hunk 1 above)
Comment (Seamus-Salt) [reply examples/use-cases/ibor/Transaction Type Consolidation.ipynb:1]: <p>Does the same hold true for Source? Might be worth mentioning something about that as well.</p>

Comment (Tomwp) [reply examples/use-cases/ibor/Transaction Type Consolidation.ipynb:1]: <p>Removed the 'Source' assignment.</p>

File: examples/use-cases/ibor/Transaction Type Consolidation.ipynb [hunk 2]
```diff
@@ -0,0 +1,1236 @@
...
```
Comment (Seamus-Salt) [inline examples/use-cases/ibor/Transaction Type Consolidation.ipynb:1]: <p>Can we show the transaction source here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/556/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (Tomwp) [reply examples/use-cases/ibor/Transaction Type Consolidation.ipynb:1]: <p>Added.</p>

**Pull request closed without merging**